app/uk/gov/hmrc/exports/config/AppConfig.scala

```diff
@@ -52,4 +58,14 @@ class AppConfig @Inject()(
 
   lazy val developerHubClientId: String =
     servicesConfig.getString("microservice.services.customs-declarations.client-id")
+
+  lazy val draftTimeToLive: FiniteDuration =
+    servicesConfig.getDuration("draft.timeToLive").asInstanceOf[FiniteDuration]
+
+  lazy val purgeDraftDeclarations: JobConfig = JobConfig(
```

Comment:
Think this needs `lifetime` or `interval` or something adding to its name

Reply:
The JobConfig val name?  How about `purgeDraftDeclarationsJob`?  it contains both the run-time and interval.

Reply:
Ignore me, just noticed it was a wrapper around two config variables